### DISCUSSION TITLE
Is there a way to see commits of a codebase in chronological order?

### DISCUSSION THREAD

**Discussion by hashiromer** [gitextensions/gitextensions#9526] [Q&A]
I would like to see how a particular codebase evolved overtime using Git Extensions, however the default order of commit history is reverse chronological order which is not appropriate for this task. 

Is there a way I could reverse the order of commit history in Git Extensions?

**Comment by pmiossec** [comment]
Maybe enter '--reverse' in the branch filter field in the toolbar.

Otherwise, simply press 'ctrl+end' to go at the beginning of the history.

**Reply by hashiromer** [reply]
I tried entering reverse in the branch filter field but it didn't work. 

'Ctrl+end' did go to the beginning of the history.
